models/articles.model.js

```diff
@@ -1,5 +1,27 @@
 const db = require('../db/connection.js')
 
+exports.checkArticleExists = (articleId) => {
+    const articleQueryString =
+        `SELECT article_id FROM articles
+    WHERE article_id = $1`
+    return db.query(articleQueryString, [articleId]).then((res) => {
+        if (res.rows.length === 0) {
+            return Promise.reject({ status: 404, msg: "article not found" })
+        }
+    })
+}
+exports.checkExists = (colName, tableName, query, msg) => {
```

Comment:
Pretty sure this is an SQL injection risk, could this be formatted differently to avoid this? May need a greenlist.